Clients/src/presentation/components/Table/index.tsx

```
@@ -152,26 +137,19 @@ const BasicTable = ({
-                    textTransform: "capitalize",
-                    borderBottom: "1px solid #EEEEEE",
-                    cursor: onRowClick ? "pointer" : "default",
-                    "&:hover": {
-                      backgroundColor: theme.palette.background.accent,
-                    },
-                  }}
+                  sx={singleTheme.tableStyles.primary.body.row}
                   key={row.id}
                   onClick={() => {
                     console.log(`Row clicked: ${row.id}`);
                     onRowClick && onRowClick(row.id);
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)